File: app/src/main/java/com/lofts/dictionary/SplashActivity.java
package com.lofts.dictionary;

import android.content.Intent;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.Window;
import android.view.WindowManager;

import com.lofts.dictionary.ui.InputActivity;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        supportRequestWindowFeature(Window.FEATURE_NO_TITLE);
        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);

        setContentView(R.layout.activity_splash);

        new Handler().postDelayed(delayRun, 3000);
    }

    Runnable delayRun = new Runnable() {
        @Override
        public void run() {
            Intent intent = new Intent();
            intent.setClass(SplashActivity.this, InputActivity.class);
            startActivity(intent);

            finish();
        }
    };

}

File: app/src/main/java/com/lofts/dictionary/SplashActivity.kt
package com.lofts.dictionary

import android.os.Bundle
import android.os.Handler
import android.support.v7.app.AppCompatActivity
import android.view.Window
import android.view.WindowManager
import com.lofts.dictionary.ui.InputActivity
import org.jetbrains.anko.intentFor
import org.jetbrains.anko.startActivity

class SplashActivity : AppCompatActivity() {

    private val delayTime: Long = 2000

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportRequestWindowFeature(Window.FEATURE_NO_TITLE)
        window.setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN)

        setContentView(R.layout.activity_splash)

        gotoIndexPage()
    }

    private fun gotoIndexPage() {
        Handler().postDelayed({
            startActivity<InputActivity>()
        }, delayTime)
    }


}
